src/components/Frequency.jsx

```diff
@@ -1,26 +1,38 @@
 import React from 'react'
 import { logarithmicScaleToFrequency } from '../math'
 import { frequencyToLogarithmicScale } from '../math'
+import { Row, Col, InputNumber, Slider } from 'antd'
+
+const addHertzLabel = value => `${value} Hz`
+const removeHertzLabel = value => value.replace(/[^\d.]/g, '')
 
 const Frequency = ({ frequency, onSetFrequency }) => {
   return (
-    <div width="100%">
-      <input
-        className="frequency-slider"
-        type="range"
-        min="0"
-        max="10"
-        step="0.0001"
-        style={{width: 700}}
-        value={frequencyToLogarithmicScale(frequency)}
-        onChange={e => onSetFrequency(logarithmicScaleToFrequency(e.target.value))}
-      />
-      <input
-        className="frequency-field"
-        value={frequency}
-        onChange={e => onSetFrequency(e.target.value)}
-      />
-    </div>
+    <Row
+      className="frequency"
+      gutter={16}
+    >
+      <Col span={3}>
+        <InputNumber
+          className="field"
+          value={frequency}
+          onChange={value => isNaN(value) ? console.log('nice try, loser!') : onSetFrequency(value)}
```

Comment:
we should remove the `console.log` here. just `!isNaN(value) && onSetFrequency(value)` no sense spamming the console for this.

Reply:
👍